src/custom/state/orders/reducer.ts

```diff
@@ -61,14 +65,31 @@ export interface PrefillStateRequired {
   chainId: ChainId
 }
 
-export type OrderTypeKeys = 'pending' | 'presignaturePending' | 'expired' | 'fulfilled' | 'cancelled'
-export const ORDER_LIST_KEYS: OrderTypeKeys[] = ['pending', 'presignaturePending', 'expired', 'fulfilled', 'cancelled']
+export type EthFlowOrderTypes = 'creating' | 'rejected' | 'refunding' | 'refunded'
+export type PreSignOrderTypes = 'presignaturePending'
+export type OrderTypeKeys = 'pending' | PreSignOrderTypes | 'expired' | 'fulfilled' | 'cancelled' | EthFlowOrderTypes
+
+export const ORDER_LIST_KEYS: OrderTypeKeys[] = [
+  'pending',
+  'presignaturePending',
+  'expired',
+  'fulfilled',
+  'cancelled',
+  'creating',
+  'rejected',
+  'refunding',
+  'refunded',
+]
 export const ORDERS_LIST: OrderLists = {
   pending: {},
   presignaturePending: {},
   fulfilled: {},
   expired: {},
   cancelled: {},
+  creating: {},
+  rejected: {},
+  refunding: {},
+  refunded: {},
 }
```

Comment:
Seriously, too many. This is a nightmare to maintain.

Too many states, mappings, transformations.
Took me quite awhile to find them all.

Reply:
Yes, we underestimated the complexity the ethflow will bring to us.

I am okay with getting rid of "refunding" state, as I am fine with showing the user only once it is refunded. But I am afraid we need another "cancelling", the txs that are created from the front-end to cancel a transaction on their own.

Reply:
To be clear @josojo this is not a jab at ethflow, but at our codebase itself :)

I added those order buckets since I was already touching the files but I'm not sure all of them will be needed.

Will revisit when I actually start with the refunding handling tasks